### PR TITLE
Misc golfs and fixes

### PR DESCRIPTION
**Pull request by RensR**

### CODE REVIEW

File: contracts/src/v0.8/ccip/offRamp/OffRamp.sol [hunk 1]
```diff
@@ -344,7 +344,7 @@ contract OffRamp is ITypeAndVersion, MultiOCR3Base {
     uint64 sourceChainSelector = report.sourceChainSelector;
     _whenNotCursed(sourceChainSelector);
 
-    SourceChainConfig storage sourceChainConfig = _getEnabledSourceChainConfig(sourceChainSelector);
```
Comment (RensR) [inline contracts/src/v0.8/ccip/offRamp/OffRamp.sol:347]: Quite large hot path gas win

File: contracts/src/v0.8/ccip/ocr/MultiOCR3Base.sol [hunk 2]
```diff
@@ -304,8 +301,8 @@ abstract contract MultiOCR3Base is ITypeAndVersion, OwnerIsCreator {
       // never have a signer role.
       Oracle memory oracle = s_oracles[ocrPluginType][signer];
       if (oracle.role != Role.Signer) revert UnauthorizedSigner();
-      if (signed[oracle.index]) revert NonUniqueSignatures();
-      signed[oracle.index] = true;
+      if (signed & (0x1 << oracle.index) != 0) revert NonUniqueSignatures();
```
Comment (RensR) [inline contracts/src/v0.8/ccip/ocr/MultiOCR3Base.sol:304]: big and hot path

Comment (RayXpub) [reply contracts/src/v0.8/ccip/ocr/MultiOCR3Base.sol:304]: nice

File: contracts/src/v0.8/ccip/offRamp/OffRamp.sol (hunk 1 above)
Comment (RayXpub) [reply contracts/src/v0.8/ccip/offRamp/OffRamp.sol:347]: Great catch

**Review by RayXpub — APPROVED** [review]

**Review by makramkd — APPROVED** [review]

**Pull request merged**